Sources/ClientRuntime/Networking/Http/SdkHttpRequest.swift

```diff
@@ -25,6 +25,18 @@ public class SdkHttpRequest {
         self.endpoint = endpoint
         self.body = body
     }
+    
+    public func toBuilder() -> SdkHttpRequestBuilder {
+        return SdkHttpRequestBuilder()
+            .withBody(self.body)
+            .withMethod(self.method)
+            .withHeaders(self.headers)
+            .withPath(self.path)
+            .withHost(self.host)
+            .withQueryItems(self.queryItems ?? [])
```

Comment:
Nil query items & empty query items are not the same thing.  We should preserve the `queryItems` value in the original request as-is.

Reply:
Changed to handle the nil `queryItems` value correctly.